### PR TITLE
plugins:foreman_cockpit - Release 1.0.2 (DEB)

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by mmoll** [comment]
as foreman_cockpit is having assets in the gem, I think this need asset precompilation

**Comment by dLobatog** [comment]
Assets precompilation added, thanks @mmoll

File: plugins/ruby-foreman-cockpit/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby-foreman-cockpit (1.0.2) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-cockpit/debian/changelog]: 1.0.2-1

File: plugins/ruby-foreman-cockpit/debian/rules [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+build:
+	cp cache/* /usr/share/foreman/vendor/cache/
+	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
```
Comment (domcleal) [inline plugins/ruby-foreman-cockpit/debian/rules]: PLUGIN isn't defined, see plugins/ruby-foreman-bootdisk/debian/rules

**Comment by dLobatog** [comment]
[test]

File: plugins/ruby-foreman-cockpit/debian/control [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Section: ruby
+Priority: extra
+Maintainer: Daniel Lobato Garcia <elobatocs@gmail.com>
+Build-Depends: debhelper, foreman-assets (>= 1.7.0), ruby-foreman-deface (<<2.0.0)
+Standards-Version: 3.9.3
+Homepage: https://github.com/theforeman/foreman_cockpit
+
+Package: ruby-foreman-cockpit
+Architecture: all
+Depends: ${misc:Depends}, bundler, ruby-foreman-deface (<<2.0.0), foreman (>= 1.7.0), foreman-assets (>= 1.7.0)
```
Comment (mmoll) [inline plugins/ruby-foreman-cockpit/debian/control]: -assets is only a build dependency

**Comment by dLobatog** [comment]
@mmoll Thanks, updated

**Comment by domcleal** [comment]
Merged as 02aad1b, thanks @dLobatog.

**Comment by domcleal** [comment]
Cherry picked onto deb/1.8 through 1.10.

**Pull request closed without merging**